src/mixins/legal-api-mixin.ts

```diff
@@ -75,4 +139,91 @@ export default class LegalApiMixin extends Vue {
         return filing
       })
   }
+
+  /**
+   * Fetches a filing's comments list.
+   * @param url the full URL to fetch the comments
+   * @returns a promise to return the comments from the response
+   */
+  async fetchComments (url: string): Promise<any> {
+    return axios.get(url)
+      .then(response => {
+        const comments = response?.data
+        if (!comments) {
+          // eslint-disable-next-line no-console
+          console.log('fetchComments() error - invalid response =', response)
+          throw new Error('Invalid API response')
+        }
+        return comments
+      })
+  }
+
+  /**
+   * Fetches a filing's documents list.
+   * @param url the full URL to fetch the documents
+   * @returns a promise to return the documents from the response
+   */
+  async fetchDocuments (url: string): Promise<any> {
+    return axios.get(url)
+      .then(response => {
+        const documents = response?.data
+        if (!documents) {
+          // eslint-disable-next-line no-console
+          console.log('fetchDocuments() error - invalid response =', response)
+          throw new Error('Invalid API response')
+        }
+        return documents
+      })
+  }
+
+  /**
+   * Fetches one document and prompts browser to open/save it.
+   * See also PayApiMixin::fetchOneReceipt().
+   * @param document the document info object
+   */
+  async fetchOneDocument (entityIncNo: string, document: any): Promise<any> {
```

Comment:
Most of these methods -- like this one -- previously existed somewhere else. I moved them here cause this is where they belong!